Modules/core/Controller/CorespaceadminController.php

```diff
@@ -243,13 +244,28 @@ public function editAction($id_space){
                 return $this->redirect("spaceadmin", [], ['space' => $newSpace]);
             }
         }
+
+         // set showTodo to true if coming back from a todo action
+         $showTodo = ($this->request->getParameterNoException('showTodo') == 1) ? true : false;
```

Comment:
plus besoin de çà du coup, normalement

Reply:
I still need that to display todolist after redirection. But don't need flash handling anymore